README.md

```diff
@@ -12,14 +12,6 @@ A suite of Koa utilities allowing for quicker bootstrapping,
 as well as a consequential guide on how to write apps using the Koa philosophy.
 Think of it as a KrakenJS for Koa.
 
-## Status
-
-Beware! Koala is alpha software!
```

Comment:
I think it would help to leave the alpha warning, unless you'd like this to go into a stable release before we publish another unstable release.